### ISSUE TITLE
any way to to extend auth validity?

### ISSUE THREAD

**Issue by gromaxbro** [MixV2/EpicResearch#144]
so i am using "grant_type": "authorization_code",
but i have to add code manually from browser is there any better way?
and more access code validity.?

**Comment by 2ir** [comment]
Yes. You can use [this](https://github.com/MixV2/EpicResearch/blob/master/docs/auth/grant_types/device_code.md)
